ng2-components/ng2-activiti-analytics/package.json

```diff
@@ -41,51 +41,36 @@
     "url": "https://github.com/Alfresco/alfresco-ng2-components/issues"
   },
   "dependencies": {
-    "@angular/common": "2.0.0",
-    "@angular/compiler": "2.0.0",
-    "@angular/core": "2.0.0",
-    "@angular/forms": "2.0.0",
-    "@angular/http": "2.0.0",
-    "@angular/platform-browser": "2.0.0",
-    "@angular/platform-browser-dynamic": "2.0.0",
-    "@angular/router": "3.0.0",
-    "@angular/upgrade": "2.0.0",
-    "@types/node": "^6.0.42",
-    "alfresco-js-api": "^0.3.0",
-    "moment": "2.15.1",
     "chart.js": "^2.1.4",
-    "core-js": "^2.4.1",
     "md-date-time-picker": "^2.2.0",
-    "ng2-alfresco-core": "0.3.2",
-    "ng2-activiti-diagrams": "0.3.3",
     "ng2-charts": "1.1.0",
-    "ng2-translate": "2.5.0",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "5.0.0-beta.12",
-    "systemjs": "0.19.27",
-    "zone.js": "^0.6.23"
+    "moment": "2.15.1",
+
+    "ng2-alfresco-core": "0.3.2",
+    "ng2-activiti-diagrams": "0.3.3"
   },
   "devDependencies": {
     "@types/core-js": "^0.9.32",
     "@types/jasmine": "^2.2.33",
     "concurrently": "^2.2.0",
-    "cpx": "^1.3.1",
-    "jasmine-ajax": "^3.2.0",
+    "cpx": "1.3.1",
     "jasmine-core": "2.4.1",
-    "karma": "~0.13.22",
-    "karma-chrome-launcher": "~1.0.1",
-    "karma-coverage": "^1.0.0",
-    "karma-jasmine": "~1.0.2",
+    "karma": "0.13.22",
+    "karma-chrome-launcher": "1.0.1",
+    "karma-coverage": "1.0.0",
+    "karma-jasmine": "1.0.2",
     "karma-jasmine-ajax": "^0.1.13",
-    "karma-jasmine-html-reporter": "^0.2.0",
-    "karma-mocha-reporter": "^2.0.3",
-    "license-check": "^1.0.4",
-    "remap-istanbul": "^0.6.3",
+    "karma-mocha-reporter": "2.0.3",
+    "karma-jasmine-html-reporter": "0.2.0",
+    "license-check": "1.1.5",
+    "remap-istanbul": "0.6.3",
     "rimraf": "2.5.2",
-    "traceur": "^0.0.91",
-    "tslint": "^3.8.1",
+    "traceur": "0.0.91",
+    "tslint": "3.15.1",
     "typescript": "^2.0.3",
-    "wsrv": "^0.1.5"
+    "wsrv": "^0.1.5",
+    "xo": "0.14.0",
+    "yargs": "4.7.0"
```

Comment:
this is not needed, yargs is a node.js command line parser library (with 14 dependencies), we don't use it directly.